erpnext/regional/doctype/import_supplier_invoice/import_supplier_invoice.json

```diff
@@ -92,13 +91,54 @@
    "label": "Upload XML Invoices"
   }
  ],
- "links": [],
- "modified": "2019-12-10 16:37:26.793398",
+ "modified": "2020-05-25 21:32:49.064579",
  "modified_by": "Administrator",
  "module": "Regional",
  "name": "Import Supplier Invoice",
  "owner": "Administrator",
- "permissions": [],
+ "permissions": [
+  {
+   "create": 1,
+   "email": 1,
+   "print": 1,
+   "read": 1,
+   "report": 1,
+   "role": "Accounts User",
+   "share": 1,
+   "write": 1
+  },
+  {
+   "email": 1,
```

Comment:
Permission for regional specific doctypes should be added in the regional setup and add a patch to add permissions for existing users. Current fix will make the doctype visible for all countries.

Refer this link for reference
https://github.com/frappe/erpnext/blob/develop/erpnext/regional/india/setup.py#L21